### PR TITLE
UPGRADE.md: improved unreleased instructions

### PR DESCRIPTION
**Pull request by simara-svatopluk**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Since improving upgrade notes from alpha5 to beta1 there were some upgrade notes left unimproved.
|New feature| No
|BC breaks| No
|Fixes issues| 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -51,17 +51,18 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 
 ## [From 7.0.0-beta1 to Unreleased]
 ### [shopsys/project-base]
-- [#497 adding php.ini to image is now done only in dockerfiles](https://github.com/shopsys/shopsys/pull/497)
+- *(optional)* [#497 adding php.ini to image is now done only in dockerfiles](https://github.com/shopsys/shopsys/pull/497)
     - you should make the same changes in your repository for the php.ini configuration files to be added to your Docker images
+        - remove all volumes from `docker-compose.yml` templates that include `php-ini-overrides.ini`
```
Comment (boris-brtan) [inline UPGRADE.md]: does this mean all `docker-compose.yml.dist` ?

File: UPGRADE.md [hunk 2]
```diff
@@ -51,17 +51,18 @@ There is a list of all the repositories maintained by monorepo, changes in log b
...
+            - remove them also from your local `docker-compose.yml`
+        - add `COPY project-base/docker/php-fpm/php-ini-overrides.ini /usr/local/etc/php/php.ini` into `docker/php-fpm/ci/Dockerfile`
```
Comment (boris-brtan) [inline UPGRADE.md]: is `project-base` in path for COPY relevant?

File: UPGRADE.md [hunk 3]
```diff
@@ -51,17 +51,18 @@ There is a list of all the repositories maintained by monorepo, changes in log b
...
     - execute `docker-compose pull` to pull new microservice images and `docker-compose up -d` to start newly pulled microservices
-    - url addresses to microservices have changed, you need to upgrade url address provided in `app/config/parameters.yml`  
+    - url addresses to microservices have changed, you need to upgrade url address provided in `app/config/parameters.yml`
         - update parameter `microservice_product_search_url` from `microservice-product-search:8000` to `microservice-product-search`
         - update parameter `microservice_product_search_export_url`, from `microservice-product-search-export:8000` to `microservice-product-search-export`
-- [#502 - fixed acceptance tests (loading DB dump)](https://github.com/shopsys/shopsys/pull/502)
-    - when you upgrade `codeception/codeception` to version `2.5.0`, you have to change parameter `populate` to `true`
-      in `tests/ShopBundle/Acceptance/acceptance.suite.yml`
+- *(optional)* when you upgrade `codeception/codeception` to version `2.5.0`, you have to change parameter `populate` to `true` in `tests/ShopBundle/Acceptance/acceptance.suite.yml`
```
Comment (boris-brtan) [inline UPGRADE.md]: is this OK?
in the change above is *(optional)* [#497 adding php.ini to image is now done only in dockerfiles
but here - [#502 - fixed acceptance tests (loading DB dump)](https://github.com/shopsys/shopsys/pull/502) disappeared.

File: UPGRADE.md [hunk 4]
```diff
@@ -72,6 +73,8 @@ There is a list of all the repositories maintained by monorepo, changes in log b
     - change dependencies:
         - `"doctrine/orm": "dev-doctrine-260-..."` -> `"shopsys/doctrine-orm": "2.6.2"`
         - `"intaro/postgres-search-bundle": "@dev"` -> `"shopsys/postgres-search-bundle": "0.1"`
+- *(optional)* speed up composer in your `php-fpm` container by adding `RUN composer global require hirak/prestissimo` into `docker/php-fpm/Dockerfile`
+- *(optional)* to enable logging of error in the `php-fpm` container, add `log_errors = true` to `docker/php-fpm/php-ini-overrides.ini`
```
Comment (boris-brtan) [inline UPGRADE.md]: of errors

File: UPGRADE.md (hunk 1 above)
Comment (simara-svatopluk) [reply UPGRADE.md]: yes, should I reformulate the sentence? Do You have suggestion maybe?

File: UPGRADE.md (hunk 3 above)
Comment (simara-svatopluk) [reply UPGRADE.md]: yes, I felt no need for the link

File: UPGRADE.md (hunk 1 above)
Comment (boris-brtan) [reply UPGRADE.md]: yes it would be better if the all of the docker-compose.yml templates end with .dist .

**Comment by boris-brtan** [comment]
great, can go to tests

File: UPGRADE.md [hunk 5]
```diff
@@ -51,17 +51,18 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 
 ## [From 7.0.0-beta1 to Unreleased]
 ### [shopsys/project-base]
-- [#497 adding php.ini to image is now done only in dockerfiles](https://github.com/shopsys/shopsys/pull/497)
+- *(optional)* [#497 adding php.ini to image is now done only in dockerfiles](https://github.com/shopsys/shopsys/pull/497)
     - you should make the same changes in your repository for the php.ini configuration files to be added to your Docker images
+        - remove all volumes from `docker-compose.yml.dist` templates that include `php-ini-overrides.ini`
+            - remove them also from your local `docker-compose.yml`
+        - add `COPY php-ini-overrides.ini /usr/local/etc/php/php.ini` into `docker/php-fpm/ci/Dockerfile`
```
Comment (PetrHeinz) [inline UPGRADE.md]: probably a typo `docker/php-fpm/ci/Dockerfile` -> `docker/php-fpm/Dockerfile`

**Pull request merged**